Take 2.1/Assets/Scripts/Controls.cs

```diff
@@ -57,13 +64,21 @@ void Update()
     private void FixedUpdate()
     {
         //flips sprite on x-axis when changing direction
-        if (inputX > 0)
-            transform.localScale = new Vector3(-1.0f, 1.0f, 1.0f);
-        if (inputX < 0)
+        if (inputX > 0.1f)
             transform.localScale = new Vector3(1.0f, 1.0f, 1.0f);
+        if (inputX < 0.1f)
+            transform.localScale = new Vector3(-1.0f, 1.0f, 1.0f);
 
         //give character velocity to move
-        rb.velocity = new Vector2(inputX * speed, rb.velocity.y);
+        if(inputX > 0.1f || inputX < -0.1f)
+        {
+          rb.AddForce(new Vector2(inputX * moveSpeed, 0f), ForceMode2D.Impulse);
+        }
+        if(!isJumping && inputY > 0.1f)
```

Comment:
need to then change isJumping to true. then change it back when it hits the ground again